wsd/LOOLWSD.cpp

```diff
@@ -580,17 +580,13 @@ class ConvertToPartHandler : public PartHandler
 {
     std::string _filename;
 
-    /// Is it really a convert-to, ie. use an especially formed path?
-    bool _convertTo;
-
 public:
     std::string getFilename() const { return _filename; }
 
     /// Afterwards someone else is responsible for cleaning that up.
     void takeFile() { _filename.clear(); }
 
-    ConvertToPartHandler(bool convertTo = false)
-        : _convertTo(convertTo)
```

Comment:
We no longer need to place convert-to files in a special directory. ConvertToBroker already flags those documents as convert-to.

Reply:
There is merit in knowing where your leaks came from - when there are different code-paths. I'd still be interested in having some source in the filename here if possible, perhaps in a follow-up